tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -424,7 +429,9 @@ describe('Filing History List - misc functionality', () => {
     expect(vm.disableCorrection({ ...item, isTypeStaff: true })).toBe(true)
 
     // only conditions[3]
-    expect(vm.disableCorrection({ ...item, isFutureEffective: true })).toBe(true)
+    for (const status of ['CANCELLED', 'DELETED', 'DRAFT', 'ERROR', 'NEW', 'PAID', 'PENDING', 'WITHDRAWN']) {
+      expect(vm.disableCorrection({ ...item, isFutureEffective: true, status })).toBe(true)
+    }
```

Comment:
Do not allow correcting a FE filing in any other state. For the record, we don't see most of these states very much in the ledger (filing history list).